packages/grid/docker-compose.pull.yml

```diff
@@ -8,15 +8,23 @@ services:
 
   seaweedfs:
     image: "${DOCKER_IMAGE_SEAWEEDFS?Variable not set}:${SEAWEEDFS_VERSION}"
+    profiles:
+      - blob-storage
 
   # docker-host:
   #   image: qoomon/docker-host
 
   proxy:
     image: ${DOCKER_IMAGE_TRAEFIK?Variable not set}:${TRAEFIK_VERSION?Variable not set}
+    profiles:
+      - proxy
 
   mongo:
     image: "${MONGO_IMAGE}:${MONGO_VERSION}"
+    profiles:
+      - mongo
 
   jaeger:
     image: jaegertracing/all-in-one:1.37
+    profiles:
```

Comment:
Since, we are currently using profiles for each container, I think ,we need to add the profile flag also to docker-compose.dev.yml?